config/crd/bases/operator.kairos.io_nodeops.yaml

```diff
@@ -148,6 +161,11 @@ spec:
                         Phase represents the current phase of the operation on this node.
                         Can be "Pending", "Running", "Completed", or "Failed"
                       type: string
+                    rebootStatus:
+                      description: |-
+                        RebootStatus represents the reboot state of this node.
+                        Can be "false" (reboot not requested), "pending" (reboot requested but not completed), or "completed" (reboot finished)
```

Comment:
how about having "not-requested" instead of false? since it's a string already, avoids confusion to think this might be a boolean and clarifies what "false" means even without reading the description.

Not a blocker

Reply:
yes, I was thinking about it too. Then I ended up also using `false` when a reboot was requested but the Job failed, in which case we don't reboot. Maybe we need 2 states instead: "not-requested" and "cancelled" or something like that. What do you think?

Reply:
yup that sounds good to me

Reply:
done